sql/moz-fx-data-shared-prod/google_ads_derived/daily_campaign_stats_v1/checks.sql

```diff
@@ -0,0 +1,2 @@
+# fail
```

Comment:
I think this needs to be `#fail` with no space (or at least all existing fail checks are written that way).
```suggestion
#fail
```